spec/models/account_status_spec.rb

```diff
@@ -13,6 +13,7 @@
 require 'spec_helper'
 require 'json'
 require 'date'
+include Freeclimb
```

Comment:
do you know why this was added?

Reply:
This was added so that we can call the module without having to specify the package from which it has come from.

So for example, instead of Freeclimb::AccountResult, we can now just call AccountResult.

Because this is generated for all spec files, we've decided to include the freeclimb package across all spec file.